charts/internal/shoot-cert-management-seed/templates/rbac.yaml

```
@@ -52,6 +52,13 @@ rules:
     - "secrets"
   verbs:
     - "*"
+- apiGroups:
+    - ""
```

Events are also available from the new event API

```suggestion
    - ""
    - events.k8s.io
```

```
$  k api-resources | grep events
events                             ev              v1                                   true         Event
events                             ev              events.k8s.io/v1                     true         Event
```

Ref https://github.com/kubernetes/enhancements/blob/master/keps/sig-instrumentation/383-new-event-api-ga-graduation/README.md